provisioning/common/scripts/k8s/diff.sh

```diff
@@ -31,12 +31,12 @@ process() {
 
   # Get ReplicaSet hashes, for example "pod-template-hash: 6749584fd9" -> "6749584fd9"
   hashes=$(cat "$in" | jq '.items[] | select(.kind == "ReplicaSet") | .metadata.labels."pod-template-hash"' --raw-output)
-  printf "Found ReplicaSet hashes in '$in' state:\n%s\n" $hashes
+  printf "Found ReplicaSet hashes in '$in' state:\n%s\n" "$hashes"
```

Comment:
Obsahuje medzeru/new line, preto musi byt v `"`, aby sa to nebralo ako viac argumentov.